contracts/lib/Errors.sol

```diff
@@ -71,297 +57,454 @@ library Errors {
+    error RoyaltyModule__ZeroAccessManager();
+
+    /// @notice Zero address provided for Dispute Module.
+    error RoyaltyModule__ZeroDisputeModule();
+
+    /// @notice Zero address provided for License Registry.
+    error RoyaltyModule__ZeroLicenseRegistry();
+
+    /// @notice Zero address provided for Licensing Module.
+    error RoyaltyModule__ZeroLicensingModule();
+
+    /// @notice Zero address provided for Royalty Policy.
     error RoyaltyModule__ZeroRoyaltyPolicy();
-    error RoyaltyModule__NotWhitelistedRoyaltyPolicy();
+
+    /// @notice Zero address provided for Royalty Token.
     error RoyaltyModule__ZeroRoyaltyToken();
+
+    /// @notice Not a whitelisted royalty policy.
+    error RoyaltyModule__NotWhitelistedRoyaltyPolicy();
+
+    /// @notice Not a whitelisted royalty token.
     error RoyaltyModule__NotWhitelistedRoyaltyToken();
+
+    /// @notice Royalty policy for IP is unset.
     error RoyaltyModule__NoRoyaltyPolicySet();
+
+    /// @notice Royalty policy between IPs are incompatible (different).
     error RoyaltyModule__IncompatibleRoyaltyPolicy();
+
+    /// @notice Caller is unauthorized.
     error RoyaltyModule__NotAllowedCaller();
-    error RoyaltyModule__ZeroLicensingModule();
+
+    /// @notice IP can only mint licenses of selected royalty policy.
     error RoyaltyModule__CanOnlyMintSelectedPolicy();
+
+    /// @notice Parent IP list for linking is empty.
     error RoyaltyModule__NoParentsOnLinking();
-    error RoyaltyModule__ZeroDisputeModule();
-    error RoyaltyModule__ZeroLicenseRegistry();
+
+    /// @notice IP is expired.
     error RoyaltyModule__IpIsExpired();
+
+    /// @notice IP is dipute tagged.
     error RoyaltyModule__IpIsTagged();
-    error RoyaltyModule__ZeroAccessManager();
 
+    ////////////////////////////////////////////////////////////////////////////
+    //                            Royalty Policy LAP                          //
+    ////////////////////////////////////////////////////////////////////////////
+
+    /// @notice Zero address provided for Access Manager in initializer.
+    error RoyaltyPolicyLAP__ZeroAccessManager();
+
+    /// @notice Zero address provided for IP Royalty Vault Beacon.
+    error RoyaltyPolicyLAP__ZeroIpRoyaltyVaultBeacon();
+
+    /// @notice Zero address provided for Royalty Module.
     error RoyaltyPolicyLAP__ZeroRoyaltyModule();
-    error RoyaltyPolicyLAP__ZeroLiquidSplitFactory();
-    error RoyaltyPolicyLAP__ZeroLiquidSplitMain();
-    error RoyaltyPolicyLAP__NotRoyaltyModule();
+
+    /// @notice Zero address provided for Licensing Module.
     error RoyaltyPolicyLAP__ZeroLicensingModule();
+
+    /// @notice Caller is not the Royalty Module.
+    error RoyaltyPolicyLAP__NotRoyaltyModule();
+
+    /// @notice Size of parent IP list is above the protocol limit.
     error RoyaltyPolicyLAP__AboveParentLimit();
+
+    /// @notice Amount of ancestors for derivative IP is above the protocol limit.
```

Comment:
Amount of ancestors for derivative IP is above the LAP royalty policy limit.

Reply:
[9736b8e](https://github.com/storyprotocol/protocol-core-v1/pull/112/commits/9736b8e55e96c43721643e61a88f491c51b900b9)